search_service/interface.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, request
+import json
+from search_service import FLASK_HOST, FLASK_PORT
+from search_service.elastic.search import create_connection, get_suggestions
+
+app = Flask(__name__)
+
+
+@app.route('/get_suggestions')
```

Comment:
the prefix `get` is redundant, because it is a GET-request. Please rename the route to `/suggestions`.